backend-rust/src/graphql_api/baker.rs

```diff
@@ -522,7 +522,7 @@ impl Baker {
                 rank,
                 total,
             }),
-            (None, None) => None,
```

Comment:
Seems pretty odd in general we can query before total lottery power has been calculated. Should probably add this to the health endpoint